Repo-Integration/Private-Registries-No-HostRules/README.md

```diff
@@ -68,15 +68,38 @@ The Auth Token environment variable must be specified in a source that allows sp
    - For the scanner, use an environment variable like:
 
      ```
-     PIP_INDEX_URL: https://<user_email>:<user_password>@<artifactory_instance>.jfrog.io/artifactory/api/pypi/default-pypi/simple
+     PIP_INDEX_URL: https://<username>:<user_password>@<artifactory_instance>.jfrog.io/artifactory/api/pypi/default-pypi/simple
      ```
 
+     If your username is an email address, then you should urlencode the "@" symbol. For ``user.example@test.org`` you would specify ``user.example%40test.org``
+
 > [!NOTE]  
 Pip prioritizes environment variables over workspace files. Refer to [https://pip.pypa.io/en/stable/topics/configuration/](https://pip.pypa.io/en/stable/topics/configuration/#precedence-override-order) for details.
 
-   No `pip.conf` file is required for the scanner or remediate container; it uses the environment variable.
+   No ``pip.conf`` file is required for the scanner or remediate container; it uses the environment variable.
+
+4. **Poetry**:
+
+   - Poetry specifies which repository it references in the ``pyproject.toml`` as well as the ``poetry.lock`` files. Therefore, the name of the repository in the pyproject.toml MUST match the name of the repository in the environment variables.  
```

Comment:
**MUST** - I feel like this should be bold if capitalized ;-)

Reply:
Fixed